NewKernel_d/include/CGAL/NewKernel_d/Types/Weighted_point.h

```diff
@@ -178,30 +178,105 @@ template <class R_> struct Power_center : Store_kernel<R_> {
     WPoint const& wp0 = *f;
     Point const& p0 = pdw(wp0);
     int d = pd(p0);
-    FT const& n0 = sdo(p0) - pw(wp0);
-    Matrix m(d,d);
-    Vec b = typename CVec::Dimension()(d);
-    // Write the point coordinates in lines.
-    int i;
-    for(i=0; ++f!=e; ++i) {
-      WPoint const& wp=*f;
-      Point const& p=pdw(wp);
-      FT const& np = sdo(p) - pw(wp);
-      for(int j=0;j<d;++j) {
-	m(i,j)=2*(c(p,j)-c(p0,j));
-	b[i] = np - n0;
+    int k = static_cast<int>(std::distance(f,e));
+    if (d+1 == k)
+    {
+      FT const& n0 = sdo(p0) - pw(wp0);
+      Matrix m(d,d);
+      Vec b = typename CVec::Dimension()(d);
+      // Write the point coordinates in lines.
+      int i;
+      for(i=0; ++f!=e; ++i) {
+	WPoint const& wp=*f;
+	Point const& p=pdw(wp);
+	for(int j=0;j<d;++j) {
+	  m(i,j)=2*(c(p,j)-c(p0,j));
+	}
+	b[i] = sdo(p) - pw(wp) - n0;
```

Comment:
Please do not indent your code with horizontal tabs. That makes the code more difficult to read in Github, and in non-conformant editors. There are tabs all over your patch.

Reply:
There were already a lot of tabs in those files, am I supposed to change them all? Use spaces only for lines I modify?

Reply:
Yes, use spaces for lines that are modified. That is short-term. For CGAL-5.1 we plan to massively convert all tabs in CGAL, and then ban them using detection scripts in our CI.